python/py-url-normalize/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-url-normalize
+version             1.4.3
+revision            0
+categories-append   devel net
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     27 36 37 38 39 310
```

Comment:
only add the subports you need as dependencies, other than that we prefer not to add "old" ones anymore